pkg/helm/chart.go

```diff
@@ -29,21 +29,43 @@ import (
 
 const ChaosMeshHelmRepo = "https://charts.chaos-mesh.org"
 
-func FetchChaosMeshChart(ctx context.Context, version string) (*chart.Chart, error) {
-	tgzPath, err := DownloadChaosMeshChartTgz(ctx, version)
-	if err != nil {
-		return nil, err
+func FetchChaosMeshChart(ctx context.Context, version, local string) (*chart.Chart, error) {
+	var (
+		tgzPath string
+		err     error
+	)
+	if local != "" {
+		if tgzPath, err = GetChaosMeshChartTgzPath(ctx, version, local); err != nil {
+			return nil, err
+		}
+	} else {
+		if tgzPath, err = DownloadChaosMeshChartTgz(ctx, version); err != nil {
+			return nil, err
+		}
 	}
+
 	requestedChart, err := loader.Load(tgzPath)
 	if err != nil {
 		return nil, errors.Wrapf(err, "load helm chart from %s", tgzPath)
 	}
 	return requestedChart, nil
 }
 
+func GetChaosMeshChartTgzPath(ctx context.Context, version, local string) (string, error) {
+	fileName := fmt.Sprintf("/chaos-mesh-%s.tgz", version)
+	tgzPath := fmt.Sprintf("%s/%s", os.TempDir(), fileName)
+	if local != "" {
+		tgzPath = fmt.Sprintf("%s/%s", local, fileName)
+	}
```

Comment:
```suggestion
	fileName := fmt.Sprintf("chaos-mesh-%s.tgz", version)
	tgzPath := fmt.Sprintf("%s/%s", os.TempDir(), fileName)
	if local != "" {
		tgzPath = fmt.Sprintf("%s/%s", local, fileName)
	}
```

the `/` already existed when concat the `tgzPath`, no need to append it in `fileName`